app/services/payment_provider_customers/stripe/retrieve_latest_payment_method_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module PaymentProviderCustomers
+  module Stripe
+    class RetrieveLatestPaymentMethodService < BaseService
+      Result = BaseResult[:payment_method_id]
+
+      def initialize(provider_customer:)
+        @provider_customer = provider_customer
+        super
+      end
+
+      def call
+        # First, we try to get the customer default payment method
+        payment_method_id = begin
+          customer = ::Stripe::Customer.retrieve(provider_customer.provider_customer_id, request_options)
+          customer["invoice_settings"]["default_payment_method"]
+        rescue
```

Comment:
I get why we are swallowing the errors, but maybe you could add a message so that it will remain clear for future readers :)